src/analyze/analyze.c

```diff
@@ -1323,7 +1323,10 @@ static int cat_config(int argc, char *argv[], void *userdata) {
 
         STRV_FOREACH(arg, argv + 1) {
                 if (arg != argv + 1)
-                        puts("");
+                        printf("%s%*s%s\n",
+                               ansi_background_blue(),
```

Comment:
hmm, so far we stayed away from changing backgrounds collors... And I'd be careful with that, after all we might end up highlighting the line too much. But the contents is not the line, but everything else really.. hence we should make sure the demarcation is there, but only suttle. i.e. visible enough but not more than that. I'd probably try to leverage the awesomeness of UTF-8 for this... i.e. something like this:

```
this is before the line
⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯⋯
and this is after the line
```

And then do that with some not too wild color, to make it sufficiently distinguishable from the rest...

(Of course one could even add a ✂ to the middle of that dotted line, but I think people would not take us seriously anymore, and who knows, people might feel urge then to cut their display in two with scissors ;-))

⋯⋯⋯⋯⋯ ✂ ⋯⋯⋯⋯⋯ ✂ ⋯⋯⋯⋯⋯ ✂ ⋯⋯⋯⋯⋯ ✂ ⋯⋯⋯⋯⋯ 

The unicode character above is not from the box drawing range, but maybe doing that would be nicer. dunno.

Reply:
Well, this has the disadvantage that copy&paste includes it. Background color has the advantage that it is ignored by pretty much everything (except people who do git too much and have acquired extra sensitivity to trailing whitespace ;)). In particular, you can take current output and pipe it to a ini-file parser and it'll just work.

> we might end up highlighting the line too much

Dunno, the worst that I can think of is that the color is too dark and the line appears black-ish on some termina / color palette. But the line has no contents anyway, so this doesn't seem to be a problem.

Reply:
> Well, this has the disadvantage that copy&paste includes it.

That is a good point, but maybe something a `#` at the beginning of the line could solve?

Reply:
Well, but then is
```
#----------------------------------------------------------------
```
more *subtle* than
``` 
                                                                       
```
? I'd say not.

Reply:
hmm, so what about outputting two empty lines, and underline the upper one? One could argue that two empty lines is a good thing anyway so that single empty lines can be used in the config files themselves, while double empty lines would then indicate where we concatenate.

Reply:
An empty underlined line also needs a block of spaces. But it looks all right. I'll push an updated version in a moment.